pkg/ir/unspecified.go

```diff
@@ -0,0 +1,95 @@
+package ir
+
+import (
+	"log"
+)
+
+func (s *Spec) ComputeBlockedEndPoints() {
+	printWarning := false
+	warningString := "The following endpoints do not have required connections; no SGs were generated for them: "
+
+	instances := make([]string, 0, len(s.Defs.InstanceToNIFs))
+	for instance := range s.Defs.InstanceToNIFs {
+		instances = append(instances, instance)
+	}
```

Comment:
In the `utils` package Elazar added a function `SortedKeys()` that takes a map and returns all its keys